docs/extensions-framework/getting-started.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Getting started"
+date: 2020-05-28T10:39:00+01:00
+weight: 1
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs
+geekdocFilePath: extensions-framework/getting-started.md
+---
+
+{{< toc >}}
+
+## Create your own extension
+- When creating a frontend for your extension, ocis-web has some requirements and options for the extension integration. Learn more about [Creating a frontend extension]({{< ref "extension-frontend.md" >}})
+
+## How to load extension into ocis-web
+All extensions are loaded when the user enters ocis-web. This is achieved with the help of [RequireJS](https://requirejs.org/).
+To load your extensions they need to be registered in the config.json.
+
+```json
+{
+  "external_apps": [
+    {
+      "id": "hello",
+      "path": "http://localhost:9105/hello.js"
+    },
+    {
+      "id": "myapp",
+      "path": "http://localhost:6789/superapp.js"
+    }
+  ]
+}
+```
+
+You can take a look at an example of a full config.json [here](https://github.com/owncloud/phoenix/blob/master/config.json.sample-ocis).
+
+## Test extension framework locally with ocis-simple
```

Comment:
`simple` doesn't exist anymore. Rewriting the first part is easy, but down in the list, steps 3 and 4 don't make sense then, because `hello` is not part of any build or deployment anymore...
- remove it from the title
- remove the first two sentences from the first paragraph
- remove the `TAGS=simple` from make